com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariables.hlsl

```diff
@@ -316,6 +321,17 @@ float2 ClampAndScaleUVForBilinear(float2 UV)
     return ClampAndScaleUV(UV, _ScreenSize.zw, 0.5f);
 }
 
+// This is assuming full screen buffer and .
```

Comment:
Just noticed I never finished typing this comment apparently (from above, I guess this is copy paste) 

`// This is assuming full screen buffer and half a texel offset for the clamping. `

Reply:
FIxed